flutter/ios/Runner/Info.plist

```diff
@@ -2,8 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CADisableMinimumFrameDurationOnPhone</key>
+	<true/>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<key>CFBundleDisplayName</key>
+	<string>Flutter Hbb</string>
```

Comment:
?

Reply:
https://github.com/rustdesk/rustdesk/blob/d9cc3433d4e4e67449c4c9388336f0ebb6cd4bbe/flutter/ios/Runner/Info.plist#L18

Reply:
When recreating ./flutter/ios, it is automatically created. I don't know why it was added.

Reply:
I removed it